tests/scripts/depends.py

```diff
@@ -421,9 +421,7 @@ def __init__(self, options):
                                           '|MBEDTLS_SHA384_'),
             # Key exchange types. Only build the library and the sample
             # programs.
-            'kex': ExclusiveDomain(key_exchange_symbols,
-                                   [build_command + ['lib'],
-                                    build_command + ['-C', 'programs']]),
+            'kex': ExclusiveDomain(key_exchange_symbols, build_and_test),
```

Comment:
Sorry for not noticing before, but the comment above is now outdated and should be removed.

Reply:
Wow, how did I miss that... Fixed, thanks!